src/video-player.js

```diff
@@ -112,6 +113,9 @@ class VideoPlayer extends BindingHelpersMixin(IocRequesterMixin(IocProviderMixin
           <template is="dom-if" if="[[hasItems(configuration.relatedVideos)]]">
             <finished-overlay state="[[state]]" related-videos="[[configuration.relatedVideos]]"></finished-overlay>
           </template>
+          <template is="dom-if" if="[[and(hasItems(configuration.quizQuestions), configuration.quizValidationCallback)]]">
```

Comment:
I don't know, I noticed that it does not work in another place, but never saw this part not working - maybe because I never really needed a recalculation. I changed it to use a computed property.